circle.yml

```diff
@@ -14,7 +14,7 @@ general:
 test:
   pre:
     - >
-      wget https://github.com/CloudSlang/cloud-slang/releases/download/cloudslang-0.9.60.0/cslang-builder.zip
+      wget https://github.com/CloudSlang/cloud-slang/releases/download/cloudslang-0.9.68/cslang-builder.zip
```

Comment:
IMHO using the latest pre-release that has `-with-content` artifact would be wiser, since it has additional validation (0.9.65)